dds/DCPS/security/AccessControl/XmlUtils.h

```diff
@@ -0,0 +1,95 @@
+/*
+ * Distributed under the OpenDDS License.
+ * See: http://www.OpenDDS.org/license.html
+ */
+
+#ifndef OPENDDS_DCPS_SECURITY_ACCESS_CONTROL_XML_UTILS_H
+#define OPENDDS_DCPS_SECURITY_ACCESS_CONTROL_XML_UTILS_H
+
+#include <dds/Versioned_Namespace.h>
+#include <dds/DCPS/SafetyProfileStreams.h>
+#include <dds/DCPS/unique_ptr.h>
+#include <dds/DCPS/security/OpenDDS_Security_Export.h>
+
+#include <dds/DdsSecurityCoreC.h>
+
+#include <ace/XML_Utils/XercesString.h>
+
+#include <xercesc/dom/DOM.hpp>
+#include <xercesc/util/XMLDateTime.hpp>
+#include <xercesc/parsers/XercesDOMParser.hpp>
+
+#include <string>
+#include <set>
+
+OPENDDS_BEGIN_VERSIONED_NAMESPACE_DECL
+
+namespace OpenDDS {
+namespace Security {
+namespace XmlUtils {
+
+typedef DCPS::unique_ptr<xercesc::XercesDOMParser> ParserPtr;
+
+OpenDDS_Security_Export
+ParserPtr get_parser(const std::string& xml, const std::string& filename);
+
+OpenDDS_Security_Export
+std::string to_string(const XMLCh* in);
+
+OpenDDS_Security_Export
+bool to_bool(const XMLCh* in, bool& value);
+
+OpenDDS_Security_Export
+bool to_time(const XMLCh* in, time_t& value);
```

Comment:
Since this pattern differs from `to_string`, should we use a different naming scheme like `parse_bool`/`parse_time`?  This may help the caller keep track of the difference between the return type and the out parameter (especially for bool).